astropy/utils/masked/tests/test_masked.py

```diff
@@ -295,40 +295,42 @@ def setup_class(self):
         super().setup_class()
         # Ensure we have used MaskedQuantity before - just in case a single test gets
         # called; see gh-15316.
-        Masked(Quantity)
+        self.MQ = Masked(Quantity)
 
-    def test_masked_quantity_class_init(self):
-        # TODO: class definitions should be more easily accessible.
+    def test_masked_quantity_getting(self):
+        # First check setup_class (or previous use) defined a cache entry.
         mcls = Masked._masked_classes[self.a.__class__]
+        # Next check this is what one gets now.
+        MQ = Masked(Quantity)
+        assert MQ is mcls
+        assert issubclass(self.MQ, Quantity)
+        assert issubclass(self.MQ, Masked)
```

Comment:
Should these be `self.` ?

Reply:
Why? `setup_class` defines `self.MQ`, but the others are imported on top.  Confused... But maybe a reason to clarify the test?

Reply:
I meant that in the above test a new `MQ = Masked(Quantity)` is created but then `self.MQ` is used in the tests instead.

Reply:
Ah, yes, that is illogical!  I'll make both changes then.